threescale_api/resources.py

```diff
@@ -637,6 +672,13 @@ def proxy(self) -> 'Proxies':
     def mapping_rules(self) -> 'MappingRules':
         return MappingRules(parent=self, instance_klass=MappingRule)
 
+    @property
+    def oidc(self):
+        return OIDCConfigs(self)
```

Comment:
here I would provide the type annotation, and also, it is not required to create the instance of the `OIDCConfigs` every time the property is called

Reply:
also comment (at least on github) to explain why this was introduced

Reply:
I've removed this